### DISCUSSION TITLE
Lucky traces

### DISCUSSION THREAD

**Discussion by ceopeo** [luckyframework/lucky#1757] [Feature requests & Ideas]
What footprints can identify technology?
If this is not possible, I suggest adding a meta tag with the version of the framework.
This will help popularize the technology and track the number of websites

**Comment by jwoertink** [comment]
Nothing identifies that you're using Lucky. I'm sort of on the fence about it. It is cool for helping spread the word with a little free "marketing", but also I think it exposes the individual sites to potential security threats should any vulnerability be discovered.

I know Kemal adds a header that can be disabled `X-Powered-By`. Maybe that's something we could do too? It would have to be easy to disable, and very clear that it's added in.... I'd love to hear more of what the community thinks on this.

**Reply by ceopeo** [reply]
Hey, thanks, yes i wanted to add your technology to the   [Wappalyzer](https://github.com/wappalyzer/wappalyzer)  project, since such a request was received from third parties.
If there is news, kick me !!!

**Reply by jwoertink** [reply]
Oh neat! I hadn't heard of that. I'm familiar with https://builtwith.com/ but pretty cool to see another type of project. I think if the community is all for it, then we can definitely add it in. Thanks for bringing this up.

**Reply by robacarp** [reply]
As long as it's something I can disable, that's fine. I don't think the default should include the lucky version.

**Reply by stephendolan** [reply]
My name is Stephen and I support this feature.

**Reply by jwoertink** [reply]
What should this look like? Do we just ship with a default [humans.txt](https://humanstxt.org/Standard.html), or use a `X-Powered-By` header, or a meta tag?

**Reply by stephendolan** [reply]
I'd definitely prefer to not have an additional file and instead just get an `X-Powered-By` header that can be disabled in a config (on by default).

**Reply by ceopeo** [reply]
> What should this look like? Do we just ship with a default [humans.txt](https://humanstxt.org/Standard.html), or use a `X-Powered-By` header, or a meta tag?

**Header** or a **Meta** tag it would be convenient for me personally ; )

**Reply by robacarp** [reply]
Yeah, as much as the humans.txt initiative is neat, this is something that belongs in a header.

**Reply by watzon** [reply]
I like the idea of an X-Powered-By header that can be disabled
